lib/parallel.rb

```diff
@@ -142,15 +167,33 @@ def unpack(data)
       producer? ? data : [@source[data], data]
     end
 
+    def stopper = @runloop_queue&.push(Stop)
```

Comment:
just to keep style consistent:
```suggestion
    def stopper
      @runloop_queue&.push(Stop)
    end
```

Reply:
also should this not be called `stop` ?

Reply:
I just used the one-line syntax for brevity, but no strong preference — happy to go with the multi-line version for consistency.

And, you're right. This `#stopper` method can be renamed to` #stop` .

Reply:
Oh, `Parallel` supports Ruby 2.7 yet, doesn't it? Ruby 2.7 doesn't support one-line method definition.
Sorry, then please fix it.